src/components/sbb-toast/sbb-toast.tsx

```diff
@@ -0,0 +1,260 @@
+
+  public connectedCallback(): void {
+    this._handlerRepository.connect();
+  }
+
+  public disconnectedCallback(): void {
+    clearTimeout(this._closeTimeout);
+    this._handlerRepository.disconnect();
+  }
+
+  private _onActionSlotChange(event: Event): void {
+    const slotNodes = (event.target as HTMLSlotElement).assignedNodes();
+
+    // Force the visual state on slotted buttons
+    slotNodes
+      .filter((el) => el.nodeName === 'SBB-BUTTON')
+      .forEach((btn: HTMLSbbButtonElement) => {
+        btn.variant = 'transparent';
+        btn.negative = true;
+        btn.size = 'm';
+      });
+
+    // Force the visual state on slotted links
+    slotNodes
+      .filter((el) => el.nodeName === 'SBB-LINK')
+      .forEach((link: HTMLSbbLinkElement) => {
+        link.variant = 'inline';
+        link.negative = true;
+      });
+  }
+
+  private _onToastAnimationEnd(event: AnimationEvent): void {
+    // On toast opened
+    if (event.animationName === 'open') {
+      this._state = 'opened';
+      this.didOpen.emit();
+
+      // Start the countdown to close it
+      if (this.timeout) {
+        this._closeTimeout = setTimeout(() => this.close(), this.timeout);
+      }
+    }
+
+    // On toast closed
+    if (event.animationName === 'close') {
+      this._state = 'closed';
+      this.didClose.emit();
+    }
+  }
+
+  /**
+   * Since we do not stack toasts, we force the closing on other existing opened toasts
+   */
+  private async _closeOtherToasts(): Promise<void> {
+    document.querySelectorAll('sbb-toast').forEach(async (t) => {
+      if ((await t.getState()) === 'opened') {
+        t.close();
+      }
+    });
+  }
+
+  public render(): JSX.Element {
+    return (
+      <Host
+        data-state={this._state}
+        data-has-icon={this._namedSlots['icon'] || !!this.iconName}
+        data-has-action={this._namedSlots['action'] || this.dismissible}
+      >
+        <div class="sbb-toast__overlay-container">
+          <div
+            class="sbb-toast"
+            aria-live={this.politeness}
+            role={this._role}
+            onAnimationEnd={(event: AnimationEvent) => this._onToastAnimationEnd(event)}
+          >
+            <div class="sbb-toast__icon">
+              <slot name="icon">{this.iconName && <sbb-icon name={this.iconName} />}</slot>
+            </div>
+
+            <div class="sbb-toast__content">
+              Test text content {/* TODO Remove this */}
+              <slot />
+            </div>
+
+            <div class="sbb-toast__action">
+              <slot name="action" onSlotchange={(event) => this._onActionSlotChange(event)}>
+                {this.dismissible && (
+                  <sbb-button
+                    class="sbb-toast__action-button"
+                    icon-name="cross-small"
+                    variant="transparent"
+                    negative={true}
+                    size="m"
+                    aria-label={i18nCloseDialog[this._currentLanguage]}
```

Comment:
not sure if i18nCloseDialog describes it best, but it's for sure not wrong :)

Reply:
The choice was between i18nCloseDialog and i18nCloseAlert. Or maybe create a new one :/
What do you think?

Reply:
I agree :-)